File: problemas/1000/1042.py
(a, b, c) = map(int, input().split(' '))

numeros = [a, b, c]
numeros.sort()

print('{}\n{}\n{}\n'.format(numeros[0], numeros[1], numeros[2]))
print('{}\n{}\n{}'.format(a, b, c))

File: problemas/1000/1042.cs
using System;

namespace uri1042
{
    internal static class Program
    {
        private static void Main()
        {
            string[] entrada = Console.ReadLine().Split(' ');
            int a = int.Parse(entrada[0]);
            int b = int.Parse(entrada[1]);
            int c = int.Parse(entrada[2]);

            int[] numeros = { a, b, c };
            Array.Sort(numeros);

            Console.WriteLine($"{numeros[0]}\n{numeros[1]}\n{numeros[2]}\n");
            Console.WriteLine($"{a}\n{b}\n{c}");
        }
    }
}